src/wkb.hpp

```diff
@@ -230,6 +214,22 @@ class writer_t
 
         return data;
     }
+
+private:
+    std::string m_data;
+    uint32_t m_srid;
+
+    std::size_t m_geometry_size_offset = 0;
+    std::size_t m_multigeometry_size_offset = 0;
+    std::size_t m_ring_size_offset = 0;
+
+    void set_size(std::size_t offset, std::size_t size)
+    {
+        assert(m_data.size() >= offset + sizeof(uint32_t));
+        auto const s = static_cast<uint32_t>(size);
+        std::memcpy(&m_data[offset], reinterpret_cast<char const *>(&s),
+                    sizeof(uint32_t));
+    }
```

Comment:
Functions should come before members.

Reply:
Fixed in the original commit. I rebased to current master and force-pushed this PR.